cli/src/test/resources/com/box/l10n/mojito/cli/command/DropExportCommandTest_IO/export/input/source/source-xliff.xliff

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xliff xmlns="urn:oasis:names:tc:xliff:document:2.0" version="2.0">
```

Comment:
looks like all other `1.2` files use `<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:okp="okapi-framework:xliff-extensions">` right now, I'd suggested reusing the same